config/settings/development.yml

```diff
@@ -1,10 +1,14 @@
 authentication:
   secret: secret
+  mode: persona
```

Comment:
This may not be necessary anymore

https://github.com/DFE-Digital/publish-teacher-training/blob/main/bin/setup#L22-L31

Reply:
Removed the setup code. :)